From 2c11d8f2f17dc53d637009e2a81817bba4066451 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:27 -0400
Subject: [fs] btrfs: remove unused variable from btrfs_new_inode

Message-id: <1407179109-13714-21-git-send-email-sandeen@redhat.com>
Patchwork-id: 86976
O-Subject: [RHEL7.1 PATCH 020/422] btrfs: remove unused variable from btrfs_new_inode
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit 99e22f783bbcd048819975b8a1463f39a9966bcf
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:22 2013 +0100

    btrfs: remove unused variable from btrfs_new_inode

    Variable owner in btrfs_new_inode is unused since commit
    d82a6f1d7e8b61ed5996334d0db66651bb43641d
    (Btrfs: kill BTRFS_I(inode)->block_group)

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/inode.c b/fs/btrfs/inode.c
index 5685d15..03ebba8 100644
--- a/fs/btrfs/inode.c
+++ b/fs/btrfs/inode.c
@@ -5388,7 +5388,6 @@ static struct inode *btrfs_new_inode(struct btrfs_trans_handle *trans,
  u32 sizes[2];
  unsigned long ptr;
  int ret;
- int owner;
 
  path = btrfs_alloc_path();
  if (!path)
@@ -5434,11 +5433,6 @@ static struct inode *btrfs_new_inode(struct btrfs_trans_handle *trans,
   */
  set_bit(BTRFS_INODE_NEEDS_FULL_SYNC, &BTRFS_I(inode)->runtime_flags);
 
- if (S_ISDIR(mode))
-  owner = 0;
- else
-  owner = 1;
-
  key[0].objectid = objectid;
  btrfs_set_key_type(&key[0], BTRFS_INODE_ITEM_KEY);
  key[0].offset = 0;
-- 
1.7.1